ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-commands.sh

```diff
@@ -50,7 +50,9 @@ echo "OO_CHANNEL:           $OO_CHANNEL"
 echo "OO_INSTALL_NAMESPACE: $OO_INSTALL_NAMESPACE"
 echo "OO_TARGET_NAMESPACES: $OO_TARGET_NAMESPACES"
 
-if [[ "$OO_INSTALL_NAMESPACE" == "!create" ]]; then
+if [[ -f "${SHARED_DIR}/cvp-common-apply-secrets-namespace.txt" ]]; then
+    OO_INSTALL_NAMESPACE=$(cat $SHARED_DIR/cvp-common-apply-secrets-namespace.txt)
```

Comment:
same suggestion as above: make the filename more generic, something like operator-install-namespace.txt so it makes sense if we/others create the namespace in different steps.

Reply:
Modified.